packages/frontend/src/pages/ViewSqlChart.tsx

```diff
@@ -144,10 +140,7 @@ const ViewSqlChart = () => {
                                             config={currentVisConfig}
                                         />
                                     )}
-                                    {(isBarChartSQLConfig(currentVisConfig) ||
-                                        isPieChartSQLConfig(
-                                            currentVisConfig,
-                                        )) &&
+                                    {!isTableChartSQLConfig(currentVisConfig) &&
```

Comment:
```suggestion
                                    {!sCartesianChartSQLConfig(currentVisConfig) &&
```

Reply:
This one, no. It's cartesian or PIE. But I can make that more explicit to make it clearer. I considered a type check for things that bottom out to echarts, but didn't do it.

Reply:
YUp, makes sense! after reading the rest of the code my suggestion is incorrect